central/imageintegration/datastore/singleton.go

```diff
@@ -22,16 +23,26 @@ var (
 	dataStore DataStore
 )
 
-func initializeDefaultIntegrations(storage store.Store) {
+func initializeIntegrations(storage store.Store) {
 	ctx := sac.WithGlobalAccessScopeChecker(context.Background(), sac.AllowAllAccessScopeChecker())
 	iis, err := storage.GetAll(ctx)
 	utils.CrashOnError(err)
+	// If we are starting from scratch in online-mode, add the default image integrations.
 	if !env.OfflineModeEnv.BooleanSetting() && len(iis) == 0 {
 		// Add default integrations
 		for _, ii := range store.DefaultImageIntegrations {
 			utils.Must(storage.Upsert(ctx, ii))
 		}
 	}
+
```

Comment:
```
if len(iis) > 0 {
  log.Infof("Attempting 3.73.1 patch fix. Starting deletion of %d image integration", len(iis))
}
```

Reply:
going to rephrase so we don't tie a comment to a release version